### PR TITLE
#156882166 chat while playing a game

### PR DESCRIPTION
**Pull request by kenoseni**
#### What does this PR do?
Add a feature that allows users chat in the game
#### Description of Task to be completed?
Players can chat while playing games, either as a user or a guest
#### How should this be manually tested?
Clone the repo and cd into it. Run *npm start* and view the app in chrome, signup and test the feature or play as guest to test the feature 
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
156882166
#### Screenshots (if appropriate)
screen shot 2018-05-14 at 10 24 04 am

#### Questions:
None

### CODE REVIEW

**Review by amarachukwu-agbo — COMMENTED** [review]
I see you have done a good job on the chat feature, well done. I suggest that you include a screenshot in the PR so that the team knows what the chat feature looks like.

**Review by Spectrumsun — COMMENTED** [review]
well done. feature working well. 👍👍

File: public/js/controllers/game.js [hunk 1]
```diff
@@ -277,6 +289,23 @@ angular.module('mean.system') //eslint-disable-line
             }
           }
         }
+
+
+        $scope.sendChatMessage = function (message) {
+          if (message) {
+            $scope.chats.$add({
+              image: game.players[game.playerIndex].avatar,
```
Comment (andela-moe) [inline public/js/controllers/game.js:297]: Are we still using _avatars_ or the _imageUrl_ for the user that is logged in?

**Review by andela-moe — COMMENTED** [review]
Please review comment.

File: public/js/controllers/game.js (hunk 1 above)
Comment (kenoseni) [reply public/js/controllers/game.js:297]: No we are not using avatars. When a user signs up, his image url is stored in this location. When he plays the game, his image is uploaded. When a guest plays the game, the avatar url is what is found at this location and used as an image.

Comment (andela-moe) [reply public/js/controllers/game.js:297]: Okay. Thank you for providing clarity

**Pull request merged**